docs/pages/guides/store-assets.mdx

```diff
@@ -0,0 +1,145 @@
+
+<ImageSpotlight
+  alt="Screenshots of the MS Office Store Listing Page"
+  src="/static/images/guides/office-screenshots.png"
+  style={{ maxWidth: 720 }}
+/>
+
+## iOS App Store Asset Requirements
+
+For the iOS App Store you can upload screenshots (images) and previews (videos). For each, Apple requires specific
+widths and heights, so make sure you reference the [documentation](https://developer.apple.com/help/app-store-connect/reference/screenshot-specifications/)
+for the exact sizing. Being even a single pixel off will mean you are unable to submit the images.
+
+See our ["Store Asset Requirements" Figma](https://www.figma.com/file/5lQcrHYhUZkIxsLYCZa3S6/Store-Asset-Requirements?type=design&node-id=1-43&mode=design&t=eFTsHStO7OY2lLsd-0) for an overview
+of the minimum asset requirements for an iOS App.
+
+### Screenshots
+
+At a minimum, Apple requires you to upload 2 screenshots for an iPhone without the notch, and 2 screenshots for the iPhone with the notch,
+but you an upload up to 10 screenshots per localisation. Note that if your app is available in multiple languages and your screenshots
+include text, you should upload screenshots with the appropriate language for each localisation.
+
+If your app runs on an iPad, you will need to also provide iPad-sized screenshots.
+
+Screenshots can be portrait or landscape.
+
+### Preview (optional)
+
+An app preview is a video you can choose to include to demonstrate how your app works. You can add up to three app previews for each screen size.
+Reference the [Apple Documentation](https://developer.apple.com/help/app-store-connect/reference/app-preview-specifications/) for a summary on
+video size and format.
+
+## Google Play Store Asset Requirements
+
+Google has specific requirements for the store asset format and dimenstions and they are different from Apple's. For the most up to
+date specifications, reference the [official documentation](https://support.google.com/googleplay/android-developer/answer/9866151) for detailed requirements
+for your Play Store assets.
+
+See our ["Store Asset Requirements" Figma](https://www.figma.com/file/5lQcrHYhUZkIxsLYCZa3S6/Store-Asset-Requirements?type=design&node-id=1-171&mode=design&t=YC6s5lYVJiwSQPki-0) for a summary
+of the minimum asset requirements for an Android App.
+
+### App Icon
+
+Unlike on the iOS App Store where the app icon is always taken automatically from the app bundle,
+on the Google Play store you need to also upload a separate App Icon for your store listing.
+
+### Feature Graphic
+
+A feature graphic must be provided in order to publish your Store Listing. It is a banner that is
+displayed at the top of your store listing page.
+
+### Screenshots
+
+You need to upload at least 4 screenshots in order to publish your app.
+
+### Video (optional)
+
+You can add one preview video to your Store Listing. The video needs to be uploaded to YouTube, and you
+add it by entering a YouTube URL in the "preview video" field.
```

Comment:
```suggestion
You can add one preview video to your Store Listing. The video needs to be uploaded to YouTube, and you can add it by entering a YouTube URL in the **preview video** field.
```

- Since it is a settings field, we usually bold them in our docs to draw attention